cli/tests/data/marathon/pods/updated_good.json

```diff
@@ -15,5 +15,10 @@
         "mem": 16.0
       }
     }
+  ],
+  "networks": [
```

Comment:
Not specifically for this field. There's a unit test that ensures we display an error message when we get a 422 back from Marathon, which is what happens when we omit this field.

I don't want to write tests that rely on the details of Marathon's validation, since they're more likely to be brittle. On the other hand, having an integration test covering the error handling code path might be a good idea, even though it's covered by a unit test already. What do you think?